cpp/independent_set/ExactRecursiveIndependentSet.cpp

```diff
@@ -0,0 +1,755 @@
+/*
+ * ExactRecrusvieIndependentSet.cpp
+ *
+ *  Created on: 26.08.2023
+ *      Author: Artur Salawa
+ */
+
+#include <independent_set/IndependentSet.hpp>
+
+#include <networkit/auxiliary/BucketPQ.hpp>
+#include <networkit/graph/GraphTools.hpp>
+
+#include <limits>
+
+namespace Koala {
+
+void RecursiveIndependentSet::run() {
+    std::vector<NetworKit::node> result = recursive();
```

Comment:
In fact, it's even better to return just this vector

Reply:
I decided to return set as in the DominatingSet